scripts/mkreleaselog

```diff
@@ -0,0 +1,233 @@
+#!/bin/zsh
```

Comment:
This is just a copy of the IPFS script, needs to be modified

Reply:
I assume this script is for pulling out the contributor list?  Do we maybe put it in its own file?

Reply:
For pulling out the contributor list and auto-generating the changelog (I think it just sniffs for "Merge pull request" commits).

Reply:
Do you mean put the contributor list in its own file? I was thinking we'd include them in release notes and the Lotus CHANGELOG, but I'm easy. That's just what IPFS does.